xonsh/environ.py

```diff
@@ -785,6 +785,10 @@ class GeneralSetting(Xettings):
         6000,
         "Number of files on the PATH above which a warning is shown.",
     )
+    COMMANDS_CACHE_SAVE_INTERMEDIATE = Var.with_default(
+        False,
+        "If enabled, the results are saved between runs and can reduce the startup time.",
```

Comment:
What is "results"?

Reply:
the binary list fetched from the function.

Reply:
I think it will be cool to have more detailed description of what will be saved i.e. "the CommandsCache will be saved between runs".

Reply:
updated